specification/storage/resource-manager/Microsoft.Storage/preview/2020-08-01-preview/storage.json

```diff
@@ -353,6 +353,94 @@
             "$ref": "./examples/DeletedAccountGet.json"
           }
         },
+        "parameters": [
+          {
+            "$ref": "#/parameters/DeletedAccountName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/SubscriptionIdParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- properties retrieved and returned successfully for the deleted account.",
+            "schema": {
+              "$ref": "#/definitions/DeletedAccountListResult"
```

Comment:
It seems the "#/definitions/DeletedAccountListResult" has "value": [], but the sample of Get doesn't have "value": [].

Reply:
The sample of GET also returns an array, but in the sample, it only contains one element. Right?

https://github.com/Azure/azure-rest-api-specs/pull/11617/files#diff-2c637bf02ea121b4df9d9a2e9f2419f0f00c73c05c8d79cfc8e65508b713fc78R11

Reply:
@allenlsy-microsoft 
I might can't quite understand your comment. The link point to the get sample in this PR.
The problem is the Get result has a "Value" tag per swagger, but result in Get sample doesn't have "Value" Tag. So seems they are not aligned.

Not sure if you have validated swagger and sample are aligned with auto rest like following before add them to PR. If not, please try it:
`autorest --azure-validator --model-validator c:\code\swagger\specification\storage\resource-manager\readme.md  --tag=package-2020-08-preview`

Reply:
The `DeletedAccountGet.json` sample has problem. Although it is named Get, it should return an array. 

I think I understand what you mean. Yes, that is a mistake. I will fix it.

Reply:
@blueww Please check whether my understanding is correct.

Reply:
For Value[], the swagger and example looks align now.
But there might be other place not match, please run autorest validation to validate they are aligned.

Reply:
Executed autorest on local. `autorest --azure-validator --model-validator .\specification\storage\resource-manager\readme.md`

I saw some warnings, but no errors. Is this fine?

For the other files, somehow it was leaked into my PR. Removed now.

Reply:
For warning, generally should be OK if not caused by your change.

For other files leak to the PR, to avoid this, a good practice is using different branch to handle different features.